crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -547,3 +568,99 @@ async fn forked_single_limit_order_test(web3: Web3) {
     assert!(sell_token_balance_before > sell_token_balance_after);
     assert!(buy_token_balance_after >= buy_token_balance_before + to_wei_with_exp(500, 6));
 }
+
+async fn forked_gnosis_single_limit_order_test(web3: Web3) {
```

Comment:
this should probably be it's own test file (gnogis.rs or so). I don't think we want to duplicate a lot of tests on gnosis chain forks. Are you planning to add any others to the core repo?

Reply:
Happy to move it into it's own file. I wasn't planning to add more, with this there are enough changes upstream for me to continue directly in our codebase and for others to easily add Gnosis e2e tests.

Reply:
I think it's fine to leave the 2 tests in the same file for now.
@squadgazzz is currently working on a forked test for 0x liquidity. Maybe he can follow up with some nicer organisation for the e2e tests.